src/cluster/pager.rs

```diff
@@ -51,14 +54,16 @@ impl<
             pager: self,
             pager_state: state,
             query,
+            qv,
+            consistency
         }
     }
 
-    pub fn query<Q>(&'a mut self, query: Q) -> QueryPager<'a, Q, SessionPager<'a, M, S, T>>
+    pub fn query<Q>(&'a mut self, query: Q, qv: Option<QueryValues>, consistency: Consistency) -> QueryPager<'a, Q, SessionPager<'a, M, S, T>>
```

Comment:
maybe we can try to come up with some another method which would allow us to provide all values which make sense from https://github.com/AlexPikalov/cdrs/blob/master/src/query/query_params_builder.rs#L6 (perhaps every except the ones related to the paging itself)

Still I'd like to have `query` with as a minimalistic API

Reply:
Ok, in the newest commit this is done

Reply:
@AlexPikalov  could you have another look? Thanks!